snap/snapcraft.yaml

```diff
@@ -35,65 +35,22 @@ apps:
 
 parts:
   bitcoin-core:
-    source: https://github.com/bitcoin/bitcoin
-    source-type: git
-    source-tag: v0.17.1
-    plugin: autotools
+    plugin: nil
     override-build: | 
-      snapcraftctl build
-      make check
-      install -m 0644 -D -t $SNAPCRAFT_PART_INSTALL/share/pixmaps share/pixmaps/bitcoin128.png
+      wget https://bitcoincore.org/bin/bitcoin-core-${SNAPCRAFT_PROJECT_VERSION}/bitcoin-${SNAPCRAFT_PROJECT_VERSION}-x86_64-linux-gnu.tar.gz
+      wget https://bitcoincore.org/bin/bitcoin-core-${SNAPCRAFT_PROJECT_VERSION}/bitcoin-${SNAPCRAFT_PROJECT_VERSION}.tar.gz
+      echo "53ffca45809127c9ba33ce0080558634101ec49de5224b2998c489b6d0fc2b17  bitcoin-${SNAPCRAFT_PROJECT_VERSION}-x86_64-linux-gnu.tar.gz" | sha256sum  --check
```

Comment:
Guess x86_64 should be substituted with the target architecture here

Reply:
Done